packages/notification/src/vaadin-notification.js

```diff
@@ -538,14 +544,16 @@ class Notification extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(Pol
 
   /** @private */
   _animatedRemoveNotificationCard() {
+    this.__cleanUpOpeningClosingState();
+
     this._card.setAttribute('closing', '');
     const name = getComputedStyle(this._card).getPropertyValue('animation-name');
     if (name && name !== 'none') {
-      const listener = () => {
+      this.__animationEndListener = () => {
         this._removeNotificationCard();
```

Comment:
nit: there's a redundant call to `this._card.removeAttribute('closing');` inside of the method now that ` __cleanUpOpeningClosingState` also does that.

Reply:
Some tests fail however if `this._card.removeAttribute('closing');` is removed from `this._removeNotificationCard();`

Reply:
Ah, I see. Probably because of the `opened` property changing in `_removeNotificationCard()`. Nevermind then.